control-plane/pkg/reconciler/kafka/kafka.go

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2022 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kafka
+
+import (
+	"fmt"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kedav1alpha1 "knative.dev/eventing-autoscaler-keda/third_party/pkg/apis/keda/v1alpha1"
+	"knative.dev/pkg/kmeta"
+	"knative.dev/pkg/reconciler"
+
+	"knative.dev/eventing-autoscaler-keda/pkg/reconciler/keda"
+	kafkainternals "knative.dev/eventing-kafka-broker/control-plane/pkg/apis/internals/kafka/eventing/v1alpha1"
+)
+
+const (
+	defaultKafkaLagThreshold = 10
+)
+
+func GenerateScaleTarget(cg *kafkainternals.ConsumerGroup) *kedav1alpha1.ScaleTarget {
+	return &kedav1alpha1.ScaleTarget{
+		Name:       cg.Name,
+		APIVersion: kafkainternals.SchemeGroupVersion.String(),
+		Kind:       "ConsumerGroup",
+	}
+}
+
+func GenerateScaleTriggers(cg *kafkainternals.ConsumerGroup, triggerAuthentication *kedav1alpha1.TriggerAuthentication) ([]kedav1alpha1.ScaleTriggers, error) {
+	triggers := []kedav1alpha1.ScaleTriggers{}
+	bootstrapServers := cg.Spec.Template.Spec.Configs.Configs["bootstrap.servers"]
```

Comment:
we have a constant for this already here: https://github.com/knative-sandbox/eventing-kafka-broker/blob/93bdae7e54da41c02b9534c54ac56df623649567/control-plane/pkg/kafka/topic.go#L33

Perhaps worth to organize this a bit ? 

(and also add one for `group.id`)?